src/systems/filecoin_vm/interpreter/vm_interpreter.go

```diff
@@ -39,45 +33,66 @@ func _withTransferFundsAssert(tree st.StateTree, from addr.Address, to addr.Addr
 	}
 }
 
-func (vmi *VMInterpreter_I) ApplyMessage(inTree st.StateTree, message msg.UnsignedMessage, minerAddr addr.Address) (
+func (vmi *VMInterpreter_I) ApplyMessage(
+	inTree st.StateTree, message msg.UnsignedMessage, messageSizeOrig int, minerAddr addr.Address) (
 	st.StateTree, msg.MessageReceipt) {
 
+	gasUsed := msg.GasAmount_Zero()
+	gasRemaining := message.GasLimit()
+
+	_applyError := func(errCode exitcode.SystemErrorCode) (st.StateTree, msg.MessageReceipt) {
+		panic("TODO: deduct gasUsed from miner block reward.")
+		panic("TODO: if used gas goes to miner anyway, is this a no-op?")
```

Comment:
Right, but this condition occurs when the message doesn't even pass the basic checks (i.e., there's no sender actor to deduct the basic used-gas fees from). In this case, my understanding was that we would charge those basic used-gas fees to the miner. That's a well-defined, but am pointing out that it's a no-op, since in the current design, all used-gas is supposed to get credited to the miner anyway.